python/lsst/obs/decam/decamMapper.py

```diff
@@ -189,10 +189,11 @@ def bypass_instcal(self, datasetType, pythonType, butlerLocation, dataId):
         header = re.sub(r'[\[](\d+)[\]]$', "[0]", instcalMap.getLocations()[0])
         md0 = afwImage.readMetadata(header)
         calib = afwImage.Calib()
-        calib.setExptime(md0.get("EXPTIME"))
         calib.setFluxMag0(10**(0.4 * md0.get("MAGZERO")))
         exp.setCalib(calib)
-        
+        visitInfo = self._makeRawVisitInfo(md0)
```

Comment:
Why the underscore on the method name?  To me, the underscore says "you should never have to call this method because it's an internal implementation detail", yet you are calling it.